localizations/zh_CN.json

```
@@ -818,104 +818,104 @@
+    "OUT_B_10": "模型 B 输出层 10",
+    "OUT_B_11": "模型 B 输出层 11",
+    "Output animation path": "动画输出路径",
     "Old unmaintained localizations that used to be a part of main repository": "曾经在主分支上但现在不再维护的旧语言包",
     "Once again, this `dynamic` demo has not removed/re-implemented all features present": "再次声明，这个选项卡现在只是个摆设，所有操作都要自己去改 CSS 文件，单击\"应用风格\"与\"应用设置\"重启后生效",
-    "Only": "只有",
-    "Only copy to models with no metadata": "仅复制到没有元数据的模型（不覆盖原有元数据）",
-    "Only copy to models with same session ID": "仅复制到具有相同 session ID 的模型",
+    "Only": "仅检索负面提示词",
+    "Only copy to models with no metadata": "仅复制到不含元数据的模型",
+    "Only copy to models with same session ID": "仅复制到具有相同 Session ID 的模型",
     "Only extract keyframes (recommended)": "只提取关键帧（推荐）",
-    "Only masked": "仅蒙版",
-    "Only masked padding, pixels": "仅蒙版模式的边缘预留像素",
+    "Only masked": "仅蒙版区域",
+    "Only masked padding, pixels": "仅蒙版区域下边缘预留像素",
     "Only show .safetensors format models": "只显示 .safetensors 格式的模型",
-    "Only show alias": "仅显示别称",
-    "Only show models that have/don't have user-added metadata": "仅显示(有/无)用户自行添加元数据的模型",
+    "Only show alias": "仅展示别名",
+    "Only show models that have/don't have user-added metadata": "按是否有元数据过滤模型",
     "Only use Random seeds (Unless comparing paths)": "只用随机种子 (除非需要对比变迁轨迹)",
-    "Only use mid-control when inference": "仅在生成图片(推理 - inference)时使用中间层控制(mid-control)",
-    "Opacity": "透明度",
+    "Only use mid-control when inference": "仅在生成图片时使用中间层控制 (mid-control)",
+    "Opacity": "不透明度",
     "Open for Clip Aesthetic!": "打开以调整 Clip 的美术风格！",
     "Open...": "打开...",
     "OpenPose Editor": "OpenPose 编辑器",
     "Optionally use [filewords] to read image captions from files.": "可选择使用 [filewords] 去从文件中读取图像的描述",
-    "Order": "排序方式",
-    "Original First": "原文在上面",
+    "Order": "排序",
+    "Original First": "原文在上",
     "Original file's hash (good for deleting duplication)": "原文件哈希(sha1算法)+输出文件后缀（方便去重）",
-    "Original filename without extension": "原文件名+输出文件后缀",
-    "Original negative prompt": "初始反向提示词",
-    "Original prompt": "初始提示词",
-    "Others": "其他",
+    "Original filename without extension": "没有扩展名的原始文件名",
+    "Original negative prompt": "原始反向提示词",
+    "Original prompt": "原始正向提示词",
+    "Others": "其它",
     "Outfill border color:": "填充颜色：",
     "Outfill method:": "填充方法：",
     "Outpainting direction": "向外绘制的方向",
     "Outpainting mk2": "向外绘制第二版",
     "Output Model Name": "输出模型文件名",
     "Output directory": "输出目录",
-    "Output directory for grids; if empty, defaults to two directories below": "宫格图的输出目录； 如果为空，则默认为以下两个目录",
-    "Output directory for images from extras tab": "附加功能选项卡的输出目录",
-    "Output directory for images; if empty, defaults to three directories below": "图像的输出目录； 如果为空，则默认为以下三个目录",
-    "Output directory for img2img grids": "图生图宫格的输出目录",
-    "Output directory for img2img images": "图生图的输出目录",
+    "Output directory for grids; if empty, defaults to two directories below": "网格图统一输出文件夹； 如果为空，则按以下两个文件夹分别输出",
```

```suggestion
    "Output directory for grids; if empty, defaults to two directories below": "宫格图统一输出目录； 如果为空，则按以下两个目录分别输出",
```
Directory - 目录  
Folder - 文件夹
Path - 路径